Workbooks/Azure Private MEC/PMEC Site Overview/PMEC Site Overview.workbook

```diff
@@ -3089,5 +3074,9 @@
       "name": "SIMs"
     }
   ],
-  "$schema": "https://raw.githubusercontent.com/microsoft/Application-Insights-Workbooks/master/schema/workbook.json"
+  "fallbackResourceIds": [
+    "/subscriptions/c5f30b9d-272b-46ea-8bf6-ec6c9c392f35/resourceGroups/APRICOT-Lab-RG-1/providers/Microsoft.MobileNetwork/mobileNetworks/apricot-pmn-1/sites/Apricot-Lab-Site"
+  ],
+  "fromTemplateId": "community-Workbooks/Azure Private MEC/PMEC Site Overview",
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json"
```

Comment:
This change shouldn't be included - again this is an artifact of the export of the workbook from the portal. (This also causes the pipeline to fail)

Reply:
Done this now.